packages/stories-block/src/block/block-types/single-story/embedControls.js

```diff
@@ -151,7 +151,11 @@ const EmbedControls = (props) => {
                       )}
                 </p>
                 {hasPoster && (
-                  <Button onClick={onRemovePoster} isTertiary>
+                  <Button
+                    style={{ margin: '0 12px' }}
```

Comment:
This button, should should have the same padding not margin as the button next to it. Is there way to make this `isPrimary` but have a white background?